pkg/backend/updates.go

```diff
@@ -64,6 +64,13 @@ const (
 	// GitHubRepo is the name of the GitHub repo, if the local git repo's remote origin is hosted on GitHub.com.
 	GitHubRepo = "github.repo"
```

Comment:
Nit: When you log that bug around switching over to only using `VCSRepoOwner` and friends, we would probably want to put a similar TODO comment above `GitHubRepo` to remove those keys when they are no longer necessary.